pkg/dns/vips_allocator.go

```diff
@@ -78,14 +78,17 @@ func (d *VIPsAllocator) CreateOrUpdateVIPConfigs() error {
 		meshes = append(meshes, mesh.GetMeta().GetName())
 	}
 
-	return d.createOrUpdateVIPConfigs(meshes...)
+	return d.createOrUpdateVIPConfigs(false, meshes...)
 }
 
 func (d *VIPsAllocator) CreateOrUpdateVIPConfig(mesh string) error {
-	return d.createOrUpdateVIPConfigs(mesh)
+	return d.createOrUpdateVIPConfigs(true, mesh)
 }
 
-func (d *VIPsAllocator) createOrUpdateVIPConfigs(meshes ...string) (errs error) {
+// createOrUpdateVIPConfigs updates VIP configs for provided meshes.
+// - fastFail - if 'true' return the first occurred error, stop processing meshes.
+//              if 'false' then process all meshes, return the list of errors
```

Comment:
I see what `failFast` does, but I don't understand why 🤔   can you help me understand?

Reply:
Just forget, I got rid of it